Src/FluentAssertions/Formatting/MultidimensionalArrayFormatter.cs

```diff
@@ -33,6 +33,8 @@ public void Format(object value, FormattedObjectGraph formattedGraph, Formatting
         int[] dimensionIndices = Enumerable.Range(0, arr.Rank).Select(dimension => arr.GetLowerBound(dimension)).ToArray();
 
         int currentLoopIndex = 0;
+
+        // ReSharper disable once NotDisposedResource (false positive)
         IEnumerator enumerator = arr.GetEnumerator();
```

Comment:
I see you reported https://youtrack.jetbrains.com/issue/QD-8114/It-is-not-a-problem-Local-variable-enumerator-is-never-disposed

It looks like https://youtrack.jetbrains.com/issue/RSRP-494839/Resharper-asks-to-dispose-IEnumerator-which-does-not-implement-IDisposable

Reply:
I'll add a link to that issue to the supression statement